requirements.txt

```diff
@@ -4,13 +4,14 @@
 #
 #    pip-compile requirements.in
 #
-absl-py==1.4.0
+absl-py==2.1.0
```

Comment:
why do we update the requirements here?

Reply:
These changes were the result of running `pip-compile requirements.in` since this is a generated file

Reply:
You are not adding new dep in `requirements.in`, plz don't update the txt file. And plz run with python3.10.